src/netspresso_trainer/utils/bbox_utils.py

```diff
@@ -0,0 +1,163 @@
+import math
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import Tensor
+from torch.fx.proxy import Proxy
+
+
+def transform_bbox(bboxes: Union[Tensor, Proxy],
+                   indicator="xywh -> xyxy",
+                   img_size: Optional[Union[int, Tuple[int, int]]]=None):
```

Comment:
Seems intended form of `img_size` is (w, h). I think it can be less confusing if change this to `shape`.

Actually, `size` and `shape` terms are not well-considered across the project. We should fix terminology as `shape` and its desired form as (h, w).

Reply:
Thank you for the review! I've included your feedback to the additional commits.
And I agree to fix the terminology as `shape` with (h, w) format.